backend/server/src/main/java/com/issuetracker/repository/IssueRepository.java

```diff
@@ -0,0 +1,149 @@
+        issues.add(new Issue(3L, milestoneInfo2, "열린 이슈 타이틀2", "열린 이슈 설명2", IssueStatus.from("open"), writer, LocalDateTime.now(), assignees, labels));
+
+        return issues;
+    }
+
+    public Issues getClosedIssues(User writer) {
+        writer = new User("test@test.com", "테스터", "http://testProfile.image.url");
+
+        Users assignees = new Users();
+        assignees.add(new User("assign@test.com", "담당자", "http://testProfile.image.url"));
+        assignees.add(new User("assign2@test.com", "담당자2", "http://testProfile.image.url"));
+
+        Labels labels = new Labels();
+        labels.add(new Label(1L, "라벨 타이틀1", "라벨 설명1", "#FF0000", "#000000"));
+        labels.add(new Label(2L, "라벨 타이틀2", "라벨 설명2", "#FF0000", "#000000"));
+        labels.add(new Label(3L, "라벨 타이틀3", "라벨 설명3", "#FF0000", "#000000"));
+
+
+        MilestoneInfo milestoneInfo = new MilestoneInfo("마일스톤 제목1", "마일스톤 내용1", LocalDateTime.now());
+
+        Issues issues = new Issues();
+        issues.add(new Issue(1L, milestoneInfo, "닫힌 이슈 타이틀1", "닫힌 이슈 설명1", IssueStatus.from("closed"), writer, LocalDateTime.now(), assignees, labels));
+
+        Milestone milestone = new Milestone(1L, issues, milestoneInfo);
+
+        return issues;
+    }
+
+    public Issues getAllIssues(User writer) {
+        writer = new User("honux@test.com", "호눅스", "http://testProfile.image.url");
+
+        Users assignees = new Users();
+        assignees.add(new User("k@test.com", "K", "http://testProfile.image.url"));
+        assignees.add(new User("pyro@test.com", "파이로", "http://testProfile.image.url"));
+
+        Labels labels = new Labels();
+        labels.add(new Label(1L, "라벨 타이틀4", "라벨 설명1", "#FF0000", "#000000"));
+        labels.add(new Label(2L, "라벨 타이틀5", "라벨 설명2", "#FF0000", "#000000"));
+        labels.add(new Label(3L, "라벨 타이틀6", "라벨 설명3", "#FF0000", "#000000"));
+
+        MilestoneInfo milestoneInfo = new MilestoneInfo("마일스톤 제목1", "마일스톤 내용1", LocalDateTime.now());
+        MilestoneInfo milestoneInfo2 = new MilestoneInfo("마일스톤 제목2", "마일스톤 내용2", LocalDateTime.now());
+
+        Issues issues = new Issues();
+        issues.add(new Issue(2L, milestoneInfo, "열린 이슈 타이틀1", "열린 이슈 설명1", IssueStatus.from("open"), writer, LocalDateTime.now(), assignees, labels));
+        issues.add(new Issue(3L, milestoneInfo2, "열린 이슈 타이틀2", "열린 이슈 설명2", IssueStatus.from("open"), writer, LocalDateTime.now(), assignees, labels));
+        issues.add(new Issue(4L, milestoneInfo2, "닫힌 이슈 타이틀1", "닫힌 이슈 설명", IssueStatus.from("closed"), writer, LocalDateTime.now(), assignees, labels));
+        issues.add(new Issue(5L, milestoneInfo2, "닫힌 이슈 타이틀2", "닫힌 이슈 설명", IssueStatus.from("closed"), writer, LocalDateTime.now(), assignees, labels));
+
+        return issues;
+
+    }
+
+
+    // 이슈의 status를 토글하는 기능을 만들고자 했음. on/off
+    // DTO 나중에 없애기
+    public void toggle(IssuesNumbers issueNumbers) {
+    }
```

Comment:
```suggestion
    // 이슈의 status를 토글하는 기능을 만들고자 했음. on/off
    // DTO 나중에 없애기
    public void toggle(Long issueId) {
    }
```

이슈를 토글할 때, 한번에 한 이슈가 아니라,
여러개를 동시에 토글해야하는 기능이 필요한지 여쭈어봅니다.
그게 아니라면, issueId 값만으로도 충분하지 않을까 하는 생각이 들어서요.

Reply:
https://github.com/ghojeong/issue-tracker/pull/43#discussion_r654555837
여기서 답변 드렸습니다!

Reply:
넵 확인했습니다